packages/captp/lib/membrane.js

```diff
@@ -0,0 +1,157 @@
+// @ts-check
+
+/**
+ * @typedef {({} | Function) & 'Near'} NearRef Intersection to make compatible only with itself.
+ * @typedef {({} | Function) & 'Far'} FarRef Intersection to make compatible only with itself.
+ */
+
+/**
+ * @param {*} x
+ */
+const IDENTITY = x => x;
+
+export const makeMembrane = (rootBlue, opts = {}) => {
```

Comment:
Since this PR no longer uses the membrane, and this membrane is known not to be in a correct state, please remote it from this PR. But put it somewhere convenient to come back to after alpha.

Reply:
Done!